example.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html ng-app="inavApp">
+
+<head>
+  <title> Star Staging by Kelly Walton</title>
+  <link rel= "stylesheet" href = "css/normalize.css">
+
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular.js"></script>
+  <script src="javascript/inavapp.js"></script>
```

Comment:
Another area in question. Your Angular app js needs to be loaded after the angular.js.